pipelines/geo-equipamientos/Jenkinsfile

```
@@ -0,0 +1,74 @@
+email = "popu-servers+jenkins@populate.tools "
+pipeline {
+    agent any
+    environment {
+        WORKING_DIR = "/tmp/geo-equipamientos"
+        PATH = "$HOME/.rbenv/shims:$PATH"
+        ETL = "/var/www/gobierto-etl-espluges/current"
+        ETL_UTILS = "/var/www/gobierto-etl-utils/current"
+        // Variables that must be defined via Jenkins UI:
+        // GOBIERTO_URL = "https://demo-datos.gobify.net"
+        // table_list = "atencio_social cultura ensenyament esplais_jubilats esport mercats residencies_geriatriques salut"
+    }
+    stages {
+        stage('Clean working dir') {
+          steps {
+            sh "cd ${ETL_UTILS}; bundle exec ruby operations/prepare-working-directory/run.rb ${WORKING_DIR}"
+          }
+        }
+        stage('Download data') {
+          steps {
+            script {
+              for (table in ${table_list}) {
+                url = "https://mun.nexusgeographics.com/geoserver/esplugues/ows?service=WFS&version=1.0.0&request=GetFeature&typeName=esplugues%3A"${table}"&maxFeatures=10000&outputFormat=csv"
+                sh '''#!/bin/bash
+                   cd ${ETL_UTILS};
+                   bundle exec ruby operations/download/run.rb  ${url} ${WORKING_DIR}/${table}_raw.csv
+                   ruby operations/load-csv-sqlite/run.rb ${WORKING_DIR}/${table}_raw.csv ${WORKING_DIR}/data.sqlite "," ${table}_raw 
+                   '''
+                    }
+            }
```

Could you try these syntax?

```suggestion
            sh '''#!/bin/bash
              for table in ${table_list}; do
                echo "Processing $table"
                cd $ETL_UTILS; ruby operations/download/run.rb "https://mun.nexusgeographics.com/geoserver/esplugues/ows?service=WFS&version=1.0.0&request=GetFeature&typeName=esplugues%3A${table}&maxFeatures=10000&outputFormat=csv" $WORKING_DIR/${table}_raw.csv
                ruby operations/load-csv-sqlite/run.rb $WORKING_DIR/${table}_raw.csv $WORKING_DIR/data.sqlite ',' ${table}_raw
              done
            '''
```